api/src/main/resources/liquibase-update-to-2.2.xml

```diff
@@ -196,7 +196,11 @@
 			<column name="diagnosis_id" type="int" autoIncrement="true">
 				<constraints primaryKey="true" nullable="false" />
 			</column>
-			<column name="encounter_id" type="int"><constraints nullable="false" /></column>
```

Comment:
Did you intentionally get rid of the encounter_id?

Reply:
I renamed it to `id` as you suggested earlier we drop the `encounter_` from the column names

Reply:
For that one i meant only the CodedOrFreeText fields